jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/JmeterDsl.java

```diff
@@ -622,6 +623,21 @@ public static DslForEachController forEachController(String name, String varsPre
   public static DslOnceOnlyController onceOnlyController(ThreadGroupChild... children) {
     return new DslOnceOnlyController(Arrays.asList(children));
   }
+  /**
+   * Builds a Runtime Controller that allows running a part of a test plan for a specified
+   * number of seconds on each iteration of a thread group.
+   *
+   *@param seconds defines the number of seconds the runtime controller should be executed on
+   *               each iteration.
+   * @param children contains the test plan elements to execute only one time on first iteration of
+   *                 each thread group.
+   * @return the controller instance for further configuration and usage.
+   * @see DslOnceOnlyController
+   */
+
+  public static DslRuntimeController runtimeController(String seconds, ThreadGroupChild... children) {
```

Comment:
Any particular reason to provide a method receiving a string value? Do you usually use JMeter expressions to specify the value? 

In general we try to provide the most specific data type for the value (in this case I think would be duration) as to ease understanding to users and avoid common problems and only provide String contract as a way to allow specifying JMeter expressions.

Reply:
The Runtime jmeter class privides the following to set the runtime:
setRuntime(long seconds)
setRuntime(String seconds)
so i thought a method to receive a String value gave the flexibility to specify an expression or a number of seconds